lib/recase.dart

```diff
@@ -1,36 +1,42 @@
 /// An instance of text to be re-cased.
 class ReCase {
   final RegExp _upperAlphaRegex = new RegExp(r'[A-Z]');
-  final RegExp _symbolRegex = new RegExp(r'[ ./_\-]');
+
+  final symbolMap = {
+    ' ': true,
+    '.': true,
+    '/': true,
+    '_': true,
+    '\\': true,
+    '-': true
+  };
 
   String originalText;
   List<String> _words;
-
   ReCase(String text) {
     this.originalText = text;
     this._words = _groupIntoWords(text);
   }
 
+  // TODO must be covered via tests
```

Comment:
Can you remove this comment? The method is private and there's no plan to expose it just for testing.

Reply:
Removed